src/main/java/seedu/address/model/person/Person.java

```diff
@@ -90,27 +76,27 @@ public boolean equals(Object other) {
         }
 
         Person otherPerson = (Person) other;
-        return name.equals(otherPerson.name)
-                && phone.equals(otherPerson.phone)
-                && email.equals(otherPerson.email)
-                && address.equals(otherPerson.address)
-                && tags.equals(otherPerson.tags);
+        return type.equals(otherPerson.type)
+                && nric.equals(otherPerson.nric)
+                && name.equals(otherPerson.name)
+                && dob.equals(otherPerson.dob)
+                && phone.equals(otherPerson.phone);
     }
```

Comment:
Good implementation for now, but we should consider what happens down the line if a person has same NRIC, Name and DOB but different phone number. Does the system treat them as different people? Ideally it would still highlight that they are the same person.